File: Column name from a given column number[3-October-2023]/Solution.cpp
//  >> CPP CODE <<

class Solution{
    public:
   string colName (long long int n)
    {
        string ans;
        while(n){
            char c = --n % 26 + 'A';
            ans = c + ans;
            n /= 26;
        }
        return ans;
    }
};

File: Column name from a given column number[3-October-2023]/Solution.java
//  >> JAVA CODE <<

class Solution
{
    String colName (long n)
    {
        StringBuffer ans=new StringBuffer("");
        
        while(n!=0)
        {
            n--;
            char ch=(char)(n%26+'A');
            ans.append(ch);
            n/=26;
        }
        
        return ans.reverse().toString();
    }
}